.github/workflows/unity.yml

```diff
@@ -0,0 +1,249 @@
+    - name: build_win64_dracoenc_unity
+      run: cmake --build build_win_64 --config MinSizeRel --target dracoenc_unity
+
+    - name: configure_win32
+      run: cmake . -G "Visual Studio 16 2019" -A Win32 -Bbuild_win_32 -DDRACO_UNITY_PLUGIN=ON -DDRACO_GLTF=ON -DDRACO_BACKWARDS_COMPATIBILITY=OFF
+    - name: build_win32
+      run: cmake --build build_win_32 --config MinSizeRel --target dracodec_unity
+
+    # Universal Windows Platform
+    - name: configure_uwp_arm
+      run: cmake . -G "Visual Studio 16 2019" -A ARM -Bbuild_uwp_arm -DDRACO_UNITY_PLUGIN=ON -DDRACO_GLTF=ON -DDRACO_BACKWARDS_COMPATIBILITY=OFF -DCMAKE_SYSTEM_NAME:String=WindowsStore -DCMAKE_SYSTEM_VERSION:String="10.0"
+    - name: build_uwp_arm
+      run: cmake --build build_uwp_arm --config MinSizeRel --target dracodec_unity
+
+    - name: configure_uwp_arm64
+      run: cmake . -G "Visual Studio 16 2019" -A ARM64 -Bbuild_uwp_arm64 -DDRACO_UNITY_PLUGIN=ON -DDRACO_GLTF=ON -DDRACO_BACKWARDS_COMPATIBILITY=OFF -DCMAKE_SYSTEM_NAME:String=WindowsStore -DCMAKE_SYSTEM_VERSION:String="10.0"
+    - name: build_uwp_arm64
+      run: cmake --build build_uwp_arm64 --config MinSizeRel --target dracodec_unity
+
+    - name: configure_uwp_x86
+      run: cmake . -G "Visual Studio 16 2019" -A Win32 -Bbuild_uwp_x86 -DDRACO_UNITY_PLUGIN=ON -DDRACO_GLTF=ON -DDRACO_BACKWARDS_COMPATIBILITY=OFF -DCMAKE_SYSTEM_NAME:String=WindowsStore -DCMAKE_SYSTEM_VERSION:String="10.0"
+    - name: build_uwp_x86
+      run: cmake --build build_uwp_x86 --config MinSizeRel --target dracodec_unity
+
+    - name: configure_uwp_x64
+      run: cmake . -G "Visual Studio 16 2019" -A x64 -Bbuild_uwp_x64 -DDRACO_UNITY_PLUGIN=ON -DDRACO_GLTF=ON -DDRACO_BACKWARDS_COMPATIBILITY=OFF -DCMAKE_SYSTEM_NAME:String=WindowsStore -DCMAKE_SYSTEM_VERSION:String="10.0"
+    - name: build_uwp_x64
+      run: cmake --build build_uwp_x64 --config MinSizeRel --target dracodec_unity
+
+    - name: package Windows
+      run: |
+        mkdir draco_win/x86
+        mkdir draco_win/x86_64
+        mkdir draco_win/WSA/ARM
+        mkdir draco_win/WSA/ARM64
+        mkdir draco_win/WSA/x86
+        mkdir draco_win/WSA/x64
+        mv build_win_32\MinSizeRel\dracodec_unity.dll draco_win/x86
+        mv build_win_64\MinSizeRel\dracodec_unity.dll draco_win/x86_64
+        mv build_win_64\MinSizeRel\dracoenc_unity.dll draco_win/x86_64
+        mv build_uwp_arm\MinSizeRel\dracodec_unity.dll draco_win/WSA/ARM
+        mv build_uwp_arm64\MinSizeRel\dracodec_unity.dll draco_win/WSA/ARM64
+        mv build_uwp_x86\MinSizeRel\dracodec_unity.dll draco_win/WSA/x86
+        mv build_uwp_x64\MinSizeRel\dracodec_unity.dll draco_win/WSA/x64
+    - name: upload artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: draco_win
+        path: draco_win
+
+
+  linux:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v1
+
+    # Probably not needed anymore with ubuntu 20.04
+    - name: install_cmake
```

Comment:
The existing draco action uses ubuntu-latest and omits this step-- I think you can drop the CMake installation.

Reply:
True.
This depends on switching to Ubuntu 20.04 or newer (conversation above), as it comes with a CMake>=3.12 (which is required)